src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -1163,7 +1163,7 @@ private CompressedPagesResult PrepareToWriteToJournal(LowLevelTransaction tx, Ab
             txHeader->CompressedSize = compressedLen;
             txHeader->UncompressedSize = totalSizeWritten;
             txHeader->PageCount = numberOfPages;
-            txHeader->Hash = Hashing.XXHash64.Calculate(compressionBuffer, (ulong)compressedLen);
+            txHeader->Hash = Hashing.XXHash64.Calculate(compressionBuffer, (ulong)compressedLen, (ulong)txHeader->TransactionId);
```

Comment:
Need to be sure to update the voron format here.
`Voron.Global.Constants.CurrentVersion`